package.json

```diff
@@ -12,6 +12,7 @@
     "lerna": "lerna",
     "lint": "lerna run --stream lint",
     "lint:fix": "lerna run lint:fix",
+    "start": "npm start --prefix packages/template-retail-react-app",
```

Comment:
It's just much nicer to be able to run `npm start` from the project root when you're dealing with changes in other packages.